definitions/marts/looker_studio_marts/ecf2/ls_ecf2_induction_period_edits.sqlx

```diff
@@ -0,0 +1,57 @@
+config {
+    type: "table",
+    assertions: {
+        uniqueKey: ["induction_period_id"]
+    },
+    bigquery: {
+        partitionBy: "DATE(update_timestamp)",
+        clusterBy: ["appropriate_body_name"]
+    },
+    description: "This mart is designed to show the updates that have been made to induction periods using the RIAB service. It uses the ecf2_teacher_induction_periods mart as a base and adds on the updates that have been made for each induction_period. It is filtered to only records which have been updated",

```

Comment:
Suggest clarifying whether updates include creations and/or deletions. I think they probably don't and that's fine but worth saying.

Reply:
Also I think there are some updates which are missed off the field_updates table like updates to updated_at - suggest you check the metadata for that and see if there's anything there to incorporate here.